gateway/crates/federated-server/src/server/graph_fetch_method.rs

```diff
@@ -65,6 +65,8 @@ impl GraphFetchMethod {
                 let gateway = gateway::generate(&federated_schema, None, config, hot_reload_config_path).await?;
 
                 sender.send(Some(Arc::new(gateway)))?;
+
+                std::mem::forget(otel_reload);
```

Comment:
The drop causes errors on stderr since the channel is closed on drop and the reload task freaks out. Becomes irrelevant with the removal of the reload layer.